Casks/red-canary-mac-monitor.rb

```diff
@@ -0,0 +1,29 @@
+cask "red-canary-mac-monitor" do
+  version "1.0.3"
+  sha256 "41a0265630b63d66af65b59f11c8ff11daffd002849e068bcf25b83223a1ef29"
+
+  url "https://github.com/redcanaryco/mac-monitor/releases/download/v#{version}/Red-Canary-Mac-Monitor-GoldCardinal-#{version.tr(".", "-")}.pkg",
+      verified: "github.com/redcanaryco/mac-monitor/"
+  name "Red Canary Mac Monitor"
+  name "red-canary-mac-monitor"
+  name "rc-mac-monitor"
+  desc "Analysis tool for security research and malware triage"
+  homepage "https://redcanary.com/mac-threat-analysis-tool/"
+
+  auto_updates true
+  depends_on macos: ">= :ventura"
+
+  pkg "Red-Canary-Mac-Monitor-GoldCardinal-#{version.tr(".", "-")}.pkg"
+
+  uninstall script: [
+    executable: "#{appdir}/Red Canary Mac Monitor.app/Contents/SharedSupport/uninstall.sh",
+    sudo:       false,
+  ]
+  uninstall pkgutil: [
+    "com.redcanary.agent",
+  ]
```

Comment:
@Brandon7CC These should be combined in to one `uninstall` stanza.

```suggestion
  uninstall pkgutil: "com.redcanary.agent",
                 script:   {
                   executable: "#{appdir}/Red Canary Mac Monitor.app/Contents/SharedSupport/uninstall.sh",
                   sudo:            false,
                 }
```

This may fail style.